msteams-platform/concepts/Cards/cards-actions.md

```diff
@@ -214,3 +214,11 @@ var button = new CardAction()
 ## signin
 
 Initiates OAuth flow, allowing bots to connect with secure services, as described in more detail here: [Authentication flow in bots](~/concepts/authentication/auth-flow-bot).
+
+## Adaptive card actions
+
+Adaptive cards support three action types.
+
+* [Action.OpenUrl](http://adaptivecards.io/explorer/Action.OpenUrl.html)
+* [Action.Submit](http://adaptivecards.io/explorer/Action.Submit.html)
```

Comment:
We need to describe the mechanics of this. For example (not an exhaustive list :))
* what kind of activity is generated?
* what fields are present?
* where do the the card input values go?